### PR TITLE
Display default model parameters (temporal and spatial) in docstrings

### PR DESCRIPTION
**Pull request by Astro-Kirsty**
Resolves issue [#4763](https://github.com/gammapy/gammapy/issues/4763)

### CODE REVIEW

File: gammapy/modeling/models/spatial.py [hunk 1]
```diff
@@ -510,6 +510,7 @@ class PointSpatialModel(SpatialModel):
     ----------
     lon_0, lat_0 : `~astropy.coordinates.Angle`
         Center position
+        Default is 0,0 deg
```
Comment (bkhelifi) [inline gammapy/modeling/models/spatial.py:513]:
```suggestion
        Default is "0 deg" , "0 deg"
```

**Review by bkhelifi — APPROVED** [review]
Thanks a lot @Astro-Kirsty

File: gammapy/modeling/models/spatial.py (hunk 1 above)
Comment (morcuended) [reply gammapy/modeling/models/spatial.py]: minor question: are the quotes needed? As all the rest of the defaults go without them

**Review by QRemy — APPROVED** [review]
Thanks @Astro-Kirsty, no comment from my side.

**Pull request merged**